.github/workflows/test-examples.yml

```diff
@@ -0,0 +1,23 @@
+name: Test Examples
+on:
+  workflow_dispatch:
```

Comment:
Would it make sense to set this as a cron job as well? This way we could still manually run it when needed, but also have it run periodically to ensure we don't forget to check the examples

Reply:
Yes good point, but I am not too sure if we will notice that is breaking. [c0122ab](https://github.com/grafana/pyroscope/pull/3773/commits/c0122ab82f38f2eda0aedba5ef76ea246eaebb06)